action.yaml

```diff
@@ -1,20 +1,22 @@
 name: 'EC-validator'
-description: 'To run ec against a particular container image built by RHTAP'
+description: To run ec against a particular container image built by RHTAP
 
 inputs:
   image:
-    description: 'The image reference'
+    description: The image reference
+    required: true
 
   key:
-    description: 'The public key for verifying signatures'
+    description: The public key for verifying signatures
     required: true
 
   policy:
-    description: 'A policy configuration file which determines what policies are to be applied'
+    description: A policy configuration file which determines what policies are to be applied
     required: true
+    deafult: github.com/enterprise-contract/config//default
```

Comment:
```suggestion
    default: github.com/enterprise-contract/config//default
```

Reply:
IIUC, once there's a default, then it's no longer required, since the user can skip it and the default value will be used. Also there's a typo fix.